src/Data/YAML.hs

```diff
@@ -118,7 +118,7 @@ import           Util
 --
 -- The diagram below depicts the standard layers of a [YAML 1.2](http://yaml.org/spec/1.2/spec.html) processor. This module covers the upper /Native/ and /Representation/ layers, whereas the "Data.YAML.Event" and "Data.YAML.Token" modules provide access to the lower /Serialization/ and /Presentation/ layers respectively.
 --
--- <<http://yaml.org/spec/1.2/overview2.png>>
+-- <<https://yaml.org/spec/1.2.2/img/overview2.svg>>
```

Comment:
Could you check whether the image is rendered as expected?

https://haskell-haddock.readthedocs.io/en/latest/markup.html#images indicates that a different syntax is required.

Reply:
Thanks for checking! No need to change it.